### ISSUE TITLE
Measure of Hatefulness

### ISSUE THREAD

**Issue by BoilerToad** [pysentimiento/pysentimiento#59]
I am trying to understand how the value of hateful content is calculated by the tool. I read the paper but I do not see a discussion of the methodology. I only see that sentiment and emotion are discussed.

Did I miss this in the write-up?  If it was not included, can you elaborate on the process?

Not having a documented process will make using your package challenging to include in a study as the peer reviewers could use the lack of details as grounds for rejecting the study.

**Comment by finiteautomata** [comment]
Dear @BoilerToad, thanks for your interest in the tool and pointing out those issues which we are very aware of. 

We haven't been able to update the paper so far (we will just update the arxiv version very soon) but in the meanwhile you can read the [RoBERTuito paper](http://www.lrec-conf.org/proceedings/lrec2022/pdf/2022.lrec-1.785.pdf) which describes the dataset we used and how we trained our models. 

TL;DR: Supposing you are using English or Spanish analyzers, we used the [hatEval](https://aclanthology.org/S19-2007/) dataset, which is centered on misogyny and racism. So, basically the output of the analyzer could be considered a score between 0 and 1 about the presence of hateful content against women and migrants. 

Let me know if you have any other questions.

**Comment by BoilerToad** [comment]
Thanks so much for your quick response.

I was reading this paper ... Birhane, Abeba, Vinay Prabhu, Sang Han, and
Vishnu Naresh Boddeti. “On Hate Scaling Laws For Data-Swamps.” arXiv, June
28, 2023. http://arxiv.org/abs/2306.13141.  They are using your package to
compare the content of hate speech inherent in large LLMs versus smaller
ones.

While thinking about utilizing your package for an effort I am looking to
study, I started digging into your paper.  Hence the questions.

I will read the new paper you sent and if I have any questions I will
contact you again.

Thanks again
Todd Firsich
PhD Student, Information Systems and Cyber Security
University of Texas in San Antonio

**Comment by finiteautomata** [comment]
@BoilerToad, we've just updated our [paper at arxiv](http://arxiv.org/abs/2106.09462). Please let me know if you have any doubt about it. 

Best,

JM
